waiter/src/waiter/process_request.clj

```diff
@@ -385,7 +387,13 @@
                               (statsd/inc! metric-group "response_bytes" unreported-bytes)
                               bytes-streamed')
                             bytes-reported-to-statsd))]
-                    (when more-bytes-possibly-available?
+                    (if-not more-bytes-possibly-available?
+                      (let [request (-> request
+                                        (utils/mark-request-time :closed))]
```

Comment:
I'm not convinced the `->` is worth it here

Reply:
Done